src/components/RepoList/style.css

```diff
@@ -0,0 +1,37 @@
+section{
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+section {
+  display: grid;
+  grid-template-columns: 30rem 30rem 30rem;
+  margin: 10px;
+}
+.card {
+
+  background-color: rgba(110, 124, 133, 0.788);
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+  padding: 10px;
+  border-style: solid;
+  border-radius: 5px; 
+  border-color: rgb(0, 0, 0);
+  height: 300px;
+
+
+  }
+
+  
+  .card:hover {
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.5);
```

Comment:
I like this hover styling, but would have been nice to have all the .card styling in the SingleRepo css file rather than spread around - makes it easier to change things probably